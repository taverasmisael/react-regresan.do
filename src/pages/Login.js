import React, { PureComponent } from 'react'
import PropTypes from 'prop-types'
import Typography from 'material-ui/Typography'
export default class LoginPage extends PureComponent {
  static propTypes = {
    prop: PropTypes.any
  }

  render() {
    return <Typography type="body1">Login</Typography>
  }
}
