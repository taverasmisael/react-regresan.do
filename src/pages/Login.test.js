import React from 'react'
import { shallow } from 'enzyme'
import Login from './Login'

describe('Pages/Login', () => {
  describe('rendering', () => {
    it('renders without crashing', () => {
      shallow(<Login />)
    })
    it('matches snapshot', () => {
      expect(shallow(<Login />)).toMatchSnapshot()
    })
  })
})
