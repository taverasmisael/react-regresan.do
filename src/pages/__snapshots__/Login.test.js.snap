// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages/Login rendering matches snapshot 1`] = `
<WithStyles(Typography)
  type="body1"
>
  Login
</WithStyles(Typography)>
`;
